src/components/History/HistoryEntry.vue

```diff
@@ -15,16 +15,20 @@
     <td class="expand text-truncate">
       {{ entry.message }}
     </td>
+    <q-tooltip v-if="$q.platform.is.mobile">
```

Comment:
No, that probably wouldn't work. From quasar docs:

> The idea is to place QTooltip inside your DOM element / component (as direct child in DOM hierarchy), when you want it to be the trigger for the QTooltip. Don’t worry about QTooltip content inheriting CSS from the container. This won’t occur, since QTooltip will be injected as a direct child of <body>.

http://quasar-framework.org/components/tooltip.html